tigerpath/templates/tigerpath/about.html

```diff
@@ -35,118 +35,34 @@ <h1 class="mb-3">About TigerPath</h1>
-          </ul>
-
-          <p class="tl-section">May 3rd: Beta Test</p>
-          <ul>
-            <li><p>The app is fully usable and useful to a user that would want to use it</p></li>
-            <li><p>Do a beta test release to the Princeton community</p></li>
-          </ul>
+    <section class="container mb-4">
+      <div class="text-section">
+        <h4>Description</h4>
+        <p>TigerPath is a web application that helps Princeton University students plan out their four-year course schedules.</p>
+        <p>There is currently no good way for students to manage what courses they’re taking and when they're taking them beyond the current or upcoming semester. Many students try to manually do this by using an Excel spreadsheet or pen-and-paper, but any mistake can easily lead to an overloaded semester or a need to take summer classes in order to fulfill missed requirements.</p>
+        <p>TigerPath aims to solve this problem by providing a platform for students to manage what courses they're taking during each semester and giving them feedback on the major requirements their schedule fulfills and does not fulfill. Our goal is for TigerPath to become a useful, enduring tool that ensures Princeton students are able to stay on top of requirements and graduate on time.</p>
+        <p>We currently fully support COS (AB and BSE), ELE, CBE, and Woody Woo, but we are continually adding more majors. If you'd like to be notified when your major is added to TigerPath, please fill out <a href="https://goo.gl/forms/pKxjmubIOSCOeR8L2">this form</a>!</p>
+      </div>
 
-          <p class="tl-section">May 9th, 10th: Demo Days</p>
-          <ul><li><p>Finish presentation</p></li></ul>
+      <div class="text-section">
+        <h4>Team Members</h4>
+        <p class="mb-0">Richard Chu - rc11@</p>
+        <p class="mb-0">Barak Nehoran - bnehoran@</p>
+        <p class="mb-0">Adeniji Ogunlana - ogunlana@</p>
+        <p class="mb-0">Daniel Leung - dl13@</p>
+      </div>
 
-          <p class="tl-section">May 13th: Project Due</p>
-          <ul><li><p>Finish final documentation</p></li></ul>
-        </div>
+      <div class="text-section">
+        <h4>Acknowledgments</h4>
+        <p class="mb-0">Jérémie Lumbroso - COS 333 Spring 2018 Advisor</p>
+        <p class="mb-0">Brian Kernighan - COS 333 Spring 2018 Professor</p>
+      </div>
 
-        <div id="about-res">
-          <h4>Resources</h4>
-          <a href="{% static 'res/tigerpath_design_doc.pdf' %}">Design Document</a>
-          <a href="https://github.com/churichard/tigerpath/raw/master/tigerpath/static/res/report.pdf">Report</a>
-          <a href="https://github.com/churichard/tigerpath/raw/master/tigerpath/static/res/productguide.pdf">Product Guide</a>
-        </div>
+      <div class="text-section">
+        <h4>Resources</h4>
+        <p class="mb-0"><a href="{% static 'res/tigerpath_design_doc.pdf' %}">Design Document</a></p>
```

Comment:
Same with these.
Should open in new tab/window if possible, since we don't want them to lose the tab with the app in it.

Reply:
Done